features/fout-meldingen.feature

```diff
@@ -0,0 +1,220 @@
+# language: nl
+Functionaliteit: Gebruiksvriendelijke en duidelijke fout meldingen
+
+    Als developer van een consumer applicatie
+    Wil ik gebruiksvriendelijke en duidelijke foutmeldingen
+    Zodat ik deze één op één kan gebruiken in de UI van mijn applicatie
+
+Rule: verplichte parameter(s) zijn opgegeven
+
+    Scenario: Verplichte parameter is niet opgegeven
+        Als '/adressen/zoek' wordt aangeroepen
+        Dan bevat de response de volgende kenmerken
+        | naam   | waarde                                         |
+        | title  | Ten minste één parameter moet worden opgegeven |
+        | status | 400                                            |
+        | detail | Verplichte parameter 'zoek' is niet opgegeven. |
+        En bevat de response de volgende invalidParams
+        | name | reason                 |
+        | zoek | Parameter is verplicht |
+
+Rule: alleen gespecificeerde parameters mogen worden gebruikt
+
+    Scenario: Niet gespecificeerde parameter
+        Als '/adressen?straat=Spui' wordt aangeroepen
+        Dan bevat de response de volgende kenmerken
+        | naam   | waarde                                                |
+        | title  | Een of meerdere parameters zijn niet correct.         |
+        | status | 400                                                   |
+        | detail | Zoeken met parameter 'straat' wordt niet ondersteund. |
+        En bevat de response de volgende invalidParams
+        | name   | reason                 |
+        | straat | wordt niet ondersteund |
+
+Rule: minimaal één optionele parameter is opgegeven
+
+    Abstract Scenario: Er zijn geen parameters opgegeven
+        Als '<path>' wordt aangeroepen
+        Dan bevat de response de volgende kenmerken
+        | naam   | waarde                                          |
+        | title  | Ten minste één parameter moet worden opgegeven. |
+        | status | 400                                             |
+        | detail | <error detail>                                  |
+        En bevat de response geen invalidParams
+
+        Voorbeelden:
```

Comment:
Betreft het hier wel voorbeelden. Volgens mij is dit juist de complete lijst met gevallen waar dit voor kan komen.